packages/client/hmi-client/src/workflow/ops/calibrate-julia/tera-calibrate-julia.vue

```diff
@@ -226,6 +285,176 @@ onMounted(() => {
 	}
 });
 
+onUnmounted(() => {
+	poller.stop();
```

Comment:
As we have a poller defined in the drill down and the node we are going to get errors when we open and close the drill down now.
By closing the drill down we are unmounting it and therefore running into a failure case.

![image](https://github.com/DARPA-ASKEM/terarium/assets/17088680/bbd7e7ee-0d4b-4316-a42d-5f71a64517a3)

Reply:
leaving this as it is for now as discussed on slack. will address in a future PR after figuring out how to handle polling if drilldown is unmounted.